### PR TITLE
Add block wrapper actions

### PR DESCRIPTION
**Pull request by qsomazzi**

### CODE REVIEW

**Comment by qsomazzi** [comment]
ping @rande

File: Resources/views/standard_layout.html.twig [hunk 1]
```diff
@@ -259,21 +259,23 @@ file that was distributed with this source code.
...
-                                                        <li class="dropdown sonata-actions">
-                                                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">{{ 'link_actions'|trans({}, 'SonataAdminBundle') }} <b class="caret"></b></a>
-                                                            <ul class="dropdown-menu" role="menu">
-                                                                {{ _actions|raw }}
-                                                            </ul>
-                                                        </li>
-                                                    {% else %}
-                                                        {{ _actions|raw }}
+                                                
+                                                {% block sonata_page_content_actions_wrappers %}
```
Comment (rande) [inline Resources/views/standard_layout.html.twig]: Can you name it sonata_admin_content_actions_wrappers ?

**Comment by OskarStark** [comment]
@qsomazzi can you describe whats the goal of this PR please?

**Comment by qsomazzi** [comment]
@OskarStark The goal of this PR is to provide an entry point if you need to override the action List. 
In my case I need to change the length split to 4 (to display more elements instead of a dropdown). 
And I think it's cleaner to override this small part than the entire block.

**Comment by OskarStark** [comment]
thank you @qsomazzi 

that makes sense

**Comment by qsomazzi** [comment]
@rande done

**Pull request merged**